src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -141,7 +141,7 @@ struct HMILevelPredicate
 
 }  // namespace
 
-#define POLICY_LIB_CHECK(return_value)                                      \
+#define POLICY_LIB_CHECK_OR_RETURN(return_value)                            \
```

Comment:
@YarikMamykinI propose to consider to name it like `POLICY_LIB_CHECK_AND_RETURN`. Because you don't check OR return `policy_manager_`. You check it anyway AND return if `policy_manager_` is invalid.

Reply:
@IGapchuk I don't agree, because return value works only when `policy_manager_` is invalid. So your version is not applicable here.